templates/institutional/institution-landing-fr.html

```diff
@@ -8,6 +8,8 @@
 ]
 dateModified: "2021-06-03"
 share: true
+feedbackPath: "/sites/feedback/ajax/report-problem-fr.html"
```

Comment:
Use a relative path instead. It won't work if the site run in a sub directory like GCWeb on github.
```suggestion
feedbackPath: "../../sites/feedback/ajax/report-problem-fr.html"
```

Reply:
Changed.Thanks